Makefile

```
@@ -41,6 +41,7 @@ FILES =		destructor.c				\
 		objects/plane/keys.c			\
 		objects/plane/plane.c			\
 		objects/plane/render.c			\
+		objects/rotate_vector.c			\
```

```suggestion
		objects/rotations.c			\
```
je pense que c'est mieux, ta fonction s'appel "rotate_object", pas de rapport direct avec les vecteurs